web-server/app/codesearch/web/controllers/SearchController.scala

```diff
@@ -36,10 +35,16 @@ trait SearchController[V <: DefaultTable] { self: InjectedController =>
     )
   }
 
-  def search(query: String, insensitive: String, precise: String, sources: String, page: String): Action[AnyContent] =
+  def search(query: String,
+             insensitive: String,
+             space: String,
+             precise: String,
+             sources: String,
+             page: String): Action[AnyContent] =
     Action.async { implicit request =>
-      val request = SearchRequest.applyRaw(query, insensitive, precise, sources, page)
-      val callURI = s"/$lang/search?query=$query&insensitive=$insensitive&precise=$precise&sources=$sources"
+      val request = SearchRequest.applyRaw(query, insensitive, space, precise, sources, page)
+      val callURI =
+        s"/$lang/search?query=$query&insensitive=$insensitive&space=$space&precise=$precise&sources=$sources"
```

Comment:
Let's rename these query parameters to `caseInsensitive` and `spaceInsensitive`. The existing ones aren't really clear.